utils/zerovec/src/varzerovec/vec.rs

```diff
@@ -405,6 +405,18 @@ impl<'a, T: VarULE + ?Sized, F: VarZeroVecFormat> VarZeroVec<'a, T, F> {
     }
 }
 
+impl<A, T, F> From<Vec<A>> for VarZeroVec<'static, T, F>
```

Comment:
not sure if we want an impliclty discarding conversion but i guess it's fine since From is known to be moving

Reply:
+1; prefer to not have the implicit moving conversion; where is it used? Seems better to change the call sites from `myvec.into()` to `(&myvec).into()` which is more explicit

Reply:
I mean, it's converting a `Vec<T>` into a `VarZeroVec<T>`, so no elements are discarded. Just because we don't move out of the `Vec` doesn't mean we cannot offer this API for ergonomics as well. But reverted.